### PR TITLE
add doc string for public method in Align

### PR DESCRIPTION
**Pull request by michiboo**
This pull request addresses issue ##1963

- [x ] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [ x] I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

- [ x] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Bio/Align/__init__.py [hunk 1]
```diff
@@ -1333,13 +1333,15 @@ class PairwiseAligner(_aligners.PairwiseAligner):
     """
 
     def align(self, seqA, seqB):
+        """Return the alignments of two sequence using PairwiseAligner."""
```
Comment (peterjc) [inline Bio/Align/__init__.py]: You need to move the "s",

```
"""Return the alignment of two sequences using PairwiseAligner."""
```

File: Bio/Align/__init__.py [hunk 2]
```diff
@@ -1333,13 +1333,15 @@ class PairwiseAligner(_aligners.PairwiseAligner):
...
         seqA = str(seqA)
         seqB = str(seqB)
         score, paths = _aligners.PairwiseAligner.align(self, seqA, seqB)
         alignments = PairwiseAlignments(seqA, seqB, score, paths)
         return alignments
 
     def score(self, seqA, seqB):
+        """Return the alignments score of two sequence using PairwiseAligner."""
```
Comment (peterjc) [inline Bio/Align/__init__.py]: """Return the alignment score of two sequences using PairwiseAligner."""

**Comment by michiboo** [comment]
Hi! It is fixed!

**Comment by peterjc** [comment]
Applied, thank you. If you want your name in the NEWS and CONTRIB file, let us know (or include this in your next pull request?).

**Pull request merged**